packages/web-components/@sfx/search-box/stories/index.ts

```diff
@@ -31,18 +31,27 @@ storiesOf('Components|Searchbox', module)
       'Default - with search and clear button',
       () => `
     <sfx-search-box searchbutton clearbutton></sfx-search-box>
+    <button @click="${this.updateTextEvent}">
```

Comment:
As mentioned for different action buttons, the ones here no longer work, presumably because `this.updateTextEvent` is undefined.